packages/client/tests/functional/query-validation/tests.ts

```diff
@@ -158,51 +164,96 @@ testMatrix.setupTestSuite(
         notAnArgument: 123,
       })
 
-      if (cliMeta.previewFeatures.includes('relationJoins')) {
+      if (relationJoinsEnabled && fullTextSearchEnabled) {
```

Comment:
`fullTextSearch` introduces a bit of a combinatorial explosion, and because we used inline snapshots now we need to handle the different cases of combinations. `fullTextSearch` introduces `UserOrderByWithRelationAndSearchRelevanceInput` vs `UserOrderByWithRelationInput`.